_submissions/module/mod-tele-new-players.md

```diff
@@ -0,0 +1,21 @@
+# Module Info
+
+- Name: TeleNewPlayers
+- Author: Skxawng-cu
+- Module TeleNewPlayers:
+  + Repository: https://github.com/skxawng-cu/mod-tele-new-players
+  + Download: https://github.com/skxawng-cu/mod-tele-new-players/archive/master.zip
+- License: AGPL
+
+# Module integration
+
+- AzerothCore hash/commit compliance: 90ed254
+- Includes configuration (.conf)?: Yes, copied by CMake
+- Includes SQL patches?: No
+- Core hooks used:
+    + PlayerScript: OnLogin, OnFirstLogin
+
+# Description
+
+This module allows you to config coords for alliance and horde to teleport the new players after the first login (Ideal for PvP servers).
```

Comment:
```suggestion
This module teleports new characters on first login to a location defined in the configuration file (ideal for PvP servers).
```

Reply:
Thanks for the suggestion. My english is not very good.